streaming/rrdpush.c

```diff
@@ -410,7 +410,7 @@ void log_stream_connection(const char *client_ip, const char *client_port, const
 
 
 static void rrdpush_sender_thread_spawn(RRDHOST *host) {
-    rrdhost_wrlock(host);
+    netdata_mutex_lock(&host->sender->mutex);
```

Comment:
You should do that for `rrdpush_sender_thread_stop()` as well.

Reply:
Yeah.... I've changed the mutex for the sender mutex, but as my helpful comment shows there could be problems in doing that.